### PR TITLE
fix: apipath fix on windows platform

### PR DESCRIPTION
**Pull request by zhazhaxia**
A Windows path problem.
[#L3](https://github.com/liximomo/lazy-compile-webpack-plugin/blob/master/lib/loaders/entry-loader.js#L3)
It should be `apiPath.replace(/\\/ig,'\\\\')`.Otherwise,the program will transform `E:\code\app.js` into `E:codeapp.js` at [ line 11](https://github.com/liximomo/lazy-compile-webpack-plugin/blob/master/lib/loaders/entry-loader.js#L11) because of the template code.
Just make a judgement it'll runs ok!

### CODE REVIEW

**Comment by liximomo** [comment]
image

File: lib/loaders/entry-loader.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
 const path = require('path');
 const loaderUtils = require('loader-utils');
-const apiPath = path.join(__dirname, 'api.js');
-
+var apiPath = path.join(__dirname, 'api.js')
+if(/win32/gi.test(process.platform)){
```
Comment (liximomo) [inline lib/loaders/entry-loader.js]: We suggest to use `loader-utils.stringifyRequest` to stringify the module path, it will cover more edge cases.

Comment (zhazhaxia) [reply lib/loaders/entry-loader.js]: Yeah, good advise ! I'll commit the code later.

Comment (zhazhaxia) [reply lib/loaders/entry-loader.js]: > I would suggest using `loaderUtils.stringifyRequest` to stringify the module path, it will cover more edge cases.

I'd tried to use [loaderUtils.stringifyRequest](https://github.com/webpack/loader-utils),the following code
```
loaderUtils.stringifyRequest(this,path.join(__dirname, 'api.js'))
// return 
// \"e:/a.js\"
```
It seems still can't resolve it.
What's your opinion?

Comment (liximomo) [reply lib/loaders/entry-loader.js]: Have you tested it in a Windows machine?

Comment (zhazhaxia) [reply lib/loaders/entry-loader.js]: > Have you tested it in a Windows machine?

yeah,i had tested it in my windows machine.

Comment (zhazhaxia) [reply lib/loaders/entry-loader.js]: > Have you tested it in a Windows machine?

The transform result on windows is `require('!!"E:/api.js"')` .
`loaderUtils.stringifyRequest` just like `JSON.stringify` to parse the string.

Comment (zhazhaxia) [reply lib/loaders/entry-loader.js]: > I would suggest using `loaderUtils.stringifyRequest` to stringify the module path, it will cover more edge cases.

Hi,I had tried another way to fix the problem. Look at the commit [fb7a4aa8a38856ab99796a2486d4c44d43af31b4](https://github.com/liximomo/lazy-compile-webpack-plugin/pull/10/commits/fb7a4aa8a38856ab99796a2486d4c44d43af31b4). It works on my windows and mac machine.

**Comment by lmiller1990** [comment]
I have the same problem. Is this repo still maintained? Or we should fork?

**Comment by Clarkkkk** [comment]
How is it going now?

**Comment by lmiller1990** [comment]
It is not going.

I think Webpack 5 has this out of the box anyway, doesn't it?

**Comment by zhazhaxia** [comment]
I had fixed it，by this [pr](https://github.com/liximomo/lazy-compile-webpack-plugin/pull/10/commits/909528ff0704317e6e80180c52f9271b6027e716) after fork the project.

**Comment by Clarkkkk** [comment]
> I had fixed it，by this [pr](https://github.com/liximomo/lazy-compile-webpack-plugin/pull/10/commits/909528ff0704317e6e80180c52f9271b6027e716) after fork the project.

@liximomo Can you merge this pr and release a new version? My project is not going to migrate to webpack 5 in the near future.
